include/eve/module/core/regular/bit_notor.hpp

```diff
@@ -49,37 +49,36 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!      template< eve::value T, eve::value Ts... >
-//!      bit_value<T, Ts...> bit_notor(T x, Ts... xs) noexcept;
+//!      // Regular overloads
+//!      constexpr auto bit_notor(value auto x, value auto ... xs)                          noexcept; // 1
+//!      constexpr auto bit_notor(kumi::non_empty_product_type auto const& tup)             noexcept; // 2
+//!
+//!      // Lanes masking
+//!      constexpr auto bit_notor[conditional_expr auto c](value auto x, value auto ... xs) noexcept; // 3
```

Comment:
This should use the /* any of th eabove overload*/ comments liek everywhere else

Reply:
perhaps but then, why only bit_notor ?

Reply:
moreover can we apply masks with tuple calls ?

Reply:
I put it there but as expalined later, it is in all functions

Reply:
however done